.github/workflows/python-package.yml

```
@@ -0,0 +1,49 @@
+name: Python package
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
```

This is in case you want one job failure not to cancel neighboring jobs:
```suggestion
    strategy:
      fail-fast: false
```
Up to you though.